external/graphdb-service.js

```diff
@@ -328,7 +328,10 @@ class GraphdbService {
             }
             return false;
         } catch (e) {
-            this.logger.error(`GraphDB not available. ${e}`);
+            this.logger.error({
+                msg: `GraphDB not available. ${e}`,
+                Event_name: 'GraphDBCheckError',
```

Comment:
Can we move all event names in constants. We should avoid using strings since it's easy to make typo when setting the event name.

Reply:
It will be implemented in [other PR](https://github.com/OriginTrail/ot-node/pull/1682).